test/install/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.16)
+project(test-sfml-install CXX)
+
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
```

Comment:
Add a comment to understand what this does

Reply:
Fixed

Reply:
I was expecting more of comment on what this actually does and less of an explanation on what to do. I now know just want to do, but no idea why that's needed.

Reply:
> I was expecting more of comment on what this actually does

That was my intent. find_package is called if this is built as a standalone project. I can rephrase it though.

Reply:
My bad, I read it as instruction "Call x to do y". I guess even now it reads more like an instruction than a comment, I'd probably expect "(This) Skips the find_package call below...", but maybe I'm just tired, so I guess it's okay

Reply:
I found a better way to write it such that it no longer sounds imperative. I'm fine taking our time to get these comments right.